### PR TITLE
Add Expo example project

### PR DESCRIPTION
**Pull request by ethanshar**
A little overview on what's in this PR
I created a new Expo project under `expoDemo`. 
I installed as a dependencies our uilib (react-native-ui-lib) and our demo app (unicorn-demo-app)
I'm using the screens from the demo app to construct an Expo demo app that is not based on react-native-navigation. 

I tested it and it works. 
There's only couple of screens that are problematic and at the moment I comment them till we figure a solution (the issue is that those screens import RNN)

### CODE REVIEW

File: expoDemo/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const Stack = createStackNavigator();
+
+export default function App() {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Main">
+        <Stack.Screen name="Main" component={MainScreen}/>
+        {_.map(screens, (screen, key) => {
+          return <Stack.Screen key={key} name={key.replace('Screen', '')} component={screen}/>;
```
Comment (M-i-k-e-l) [inline expoDemo/App.js]: Not sure why, but I'm not seeing anything except the title ("Main") (on iOS simulator).
Maybe we can look at it together.
BTW, I cannot connect with a real device (Android or iOS), but I'm guessing it's a routing issues.

Comment (ethanshar) [reply expoDemo/App.js]: fixed

File: expoDemo/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.expodemo">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+
+    <!-- OPTIONAL PERMISSIONS, REMOVE WHATEVER YOU DO NOT NEED -->
+    <uses-permission android:name="android.permission.MANAGE_DOCUMENTS" />
```
[the remark this answers is not in the document]
Comment (ethanshar) [reply expoDemo/android/app/src/main/AndroidManifest.xml]: That's the initial project that comes when installing Expo. 
I don't see any reason to change anything here.. this app is not meant to go to store, also it's open source, users can do whatever..

File: expoDemo/ios/Podfile.lock [hunk 3]
```diff
@@ -0,0 +1,538 @@
+PODS:
```
[the remark this answers is not in the document]
Comment (ethanshar) [reply expoDemo/ios/Podfile.lock]: No reason to change or remove for now..

**Pull request merged**